### ISSUE TITLE
sopt.minimize

### ISSUE THREAD

**Issue by mengxzh15** [brunorigal/autograd-minimize#2]
why not i use the sopt.minimize directly? I look for minimize with pytorch version, not for this.

**Comment by brunorigal** [comment]
I am not sure I understand your question. 

Are you referring to the scipy.optimize.minimize function? The advantage of this library is to use the automatic gradient capabilities of pytorch.

**Comment by brunorigal** [comment]
Or maybe you are referring to this library: https://github.com/Lyrichu/sopt
The advantage of autograd-minimize is the same: by using exact gradient computation (from pytorch or tensorflow), the gradient is faster to compute more precise and thus better suited for second order optimization methods.
